### ISSUE TITLE
Share checks with starcheck?

### ISSUE THREAD

**Issue by jeanconn** [sot/aca_preview#4]
aca_preview is awesome, but should we brainstorm on ways we could actually share checks between starcheck and this code?  Or is duplication in this case totally reasonable given the architectures of each?

**Comment by taldcroft** [comment]
Good idea, and quite straightforward.  Basically you just need to generate an `ACAReviewTable` from the backstop catalog (with includes etc), then call the check methods and look for messages.  From starcheck (Perl) it would be necessary to verify that the proseco catalog is exactly the same as the starcheck catalog before trusting `aca_preview` results.

This starts me thinking about moving the checks back into proseco.  There's nothing preventing running the checks (optionally) in ORviewer and giving a very early heads-up of issues.

Thinking even further ahead, integrating the proseco-level catalog checks into hopper would be good. That might actually be easy as well by using the proseco `ACACatalogTable` classes for representing the catalog.

Wow, this could be good!!

**Comment by jeanconn** [comment]
Speaking of shared checks, we could use a new shared data product to capture bad stars and updated catalog information for specific stars and/or processed star histories.

**Comment by taldcroft** [comment]
See https://github.com/sot/ska-projects/issues/92 and start adding ideas there.
